src/accounts-service-access.vala

```diff
@@ -210,10 +208,14 @@ public class AccountsServiceAccess : Object
             warning ("unable to sync last running player %s to AccountsService: %s",last_running_player, e.message);
         }
         _last_running_player = last_running_player;
+#else
+        return;
+#endif
     }
 
     private async void sync_volume_to_accountsservice (double volume)
     {
+#if HAS_LOMIRI_SOUND_SCHEMA
```

Comment:
ifdef HAS_LOMIRI_ACCTSERVICE_SOUND_SETTINGS

Reply:
Ignoring, as we agreed to simplify this in the next PR.